packages/airnode-validator/src/ois/ois.ts

```diff
@@ -37,10 +37,13 @@ export const fixedParameterSchema = z
 
 export const methodSchema = z.union([z.literal('get'), z.literal('post')]);
 
+// Path name must start wih "/" and must not contain space character
+export const pathNameSchema = z.string().regex(/^\/[^\s]+$/);
```

Comment:
The path can be just a `/`, right? So, the regex should be `/^\/[^\s]*$/`. Adding a test for this edge case might also be a good idea.

Reply:
Can really anything but whitespaces be in the path? What about another `/`? And random characters? Is `/@#%/%&*"\` a correct path?

Reply:
Probably not, but I wasn't sure what exactly is allowed (found conflicting information on the web) so I only disallowed spaces. 

Good catch with only a path.

Reply:
But wouldn't it be better to be more strict? We're using the `path` to build an HTTP request, right? Imagine a user passing something weird to the path that is not supported. I would prefer to catch that during the validation which will give us a meaningful message and we will immediately know what's the problem instead of letting it fail in Axios most likely without any meaningful error message.

Reply:
What regex do you suggest? For reference these are the links I found:
- https://stackoverflow.com/questions/11469882/validate-the-path-of-an-url
- https://stackoverflow.com/questions/12928781/regex-validate-a-url-path-with-no-query-params
- https://stackoverflow.com/questions/161738/what-is-the-best-regular-expression-to-check-if-a-string-is-a-valid-url

I'd like to prevent a scenario where we make an error out of a valid URL, since the API provider will not be able to deploy his Airnode. For that reason I am more inclined to less bulletproof validation.

Reply:
Ok, fair point. Looking at those regexes it can get out of hand pretty quickly. Let's let it looser and we can change it if it becomes a problem.